applications/pusher/src/pusher_maintenance.erl

```diff
@@ -10,18 +10,27 @@
 
 -include("pusher.hrl").
 
--export([add_google_app/2, add_apple_app/2]).
+-export([add_google_app/2
+        ,add_apple_app/2, add_apple_app/3
+        ]).
 
 -spec add_google_app(binary(), binary()) -> 'ok'.
 add_google_app(AppId, Secret) ->
     kapps_config:set_node(?CONFIG_CAT, <<"google">>, Secret, AppId),
     'ok'.
 
 -spec add_apple_app(binary(), binary()) -> 'ok' | {'error', any()}.
+-spec add_apple_app(binary(), binary(), binary()) -> 'ok' | {'error', any()}.
 add_apple_app(AppId, Certfile) ->
+    add_apple_app(AppId, Certfile, ?DEFAULT_APNS_HOST).
+
+add_apple_app(AppId, Certfile, Host) ->
     case file:read_file(Certfile) of
         {'ok', Binary} ->
-            kapps_config:set_node(?CONFIG_CAT, <<"apple">>, Binary, AppId),
+            Config = kz_json:from_list([{<<"certificate">>, Binary}
+                                       ,{<<"host">>, Host}
+                                       ]),
+            kapps_config:set_node(?CONFIG_CAT, <<"apple">>, Config, AppId),
```

Comment:
I would also prefer you setting these using multiple calls to `set_node` (using a path like `[<<"apple">>,<<"host">>]` in the second argument.